src/plasma/test/test.py

```diff
@@ -101,6 +102,24 @@ def test_create(self):
     for i in range(length):
       self.assertEqual(memory_buffer[i], chr(i % 256))
 
+  def test_persist_sanity(self):
```

Comment:
once you implemented the reload if the object doesn't exist already, please also write a test case for this.